src/Command/CleanMediaCommand.php

```diff
@@ -32,7 +31,7 @@ final class CleanMediaCommand extends Command
     protected static $defaultDescription = 'Find orphaned files in media upload directory';
 
     /**
-     * @var MediaProviderInterface[]|null
+     * @var string[]|null
```

Comment:
Are these strings service ids or FQCNs? Just wondering if we could use `class-string`.

Reply:
They are strings, `$provider->getName()`